revealable.css

```diff
@@ -0,0 +1,46 @@
+.revealable {
```

Comment:
This CSS is meant to be agnostic to the toolkit.

I'm not sure there is any such thing as canonical CSS types, unless you're talking in the context of things like bootstrap. I couldn't find these concepts in bootstrap.